src/main/java/com/zufar/onlinestore/cart/endpoint/CartEndpoint.java

```diff
@@ -30,14 +31,14 @@
 @RequiredArgsConstructor
 @Validated
 @RequestMapping(value = CartEndpoint.CART_URL)
-public class CartEndpoint {
+public class CartEndpoint implements CartsApi {
 
     public static final String CART_URL = "/api/v1/cart";
 
     private final CartApi cartApi;
 
     @PostMapping(value = "/items")
```

Comment:
we need to  @Override the method

Reply:
They are overrided automaticly, isn't it?

Reply:
using this annotation is like a best practice